OpenRA.Mods.Common/Projectiles/AreaBeam.cs

```diff
@@ -82,9 +82,13 @@ public class AreaBeam : IProjectile, ISync
 		readonly Color color;
 		readonly WDist speed;
 
-		[Sync] WPos headPos;
-		[Sync] WPos tailPos;
-		[Sync] WPos target;
+		[Sync]
+		WPos headPos;
+		[Sync]
+		WPos tailPos;
+		[Sync]
+		WPos target;
```

Comment:
In most other places you put a space between these.

From a consistency perspective I guess that should be done here, too, though I don't consider it a blocker.